src/lib/dnssd/Discovery_ImplPlatform.cpp

```diff
@@ -75,8 +85,9 @@ static void HandleNodeBrowse(void * context, DnssdService * services, size_t ser
 
         auto & ipAddress = services[i].mAddress;
 
-        // Check if SRV, TXT and AAAA records were received in DNS responses
-        if (strlen(services[i].mHostName) == 0 || services[i].mTextEntrySize == 0 || !ipAddress.has_value())
+        // Check that this is not operational browse result and if SRV, TXT and AAAA records were received in DNS responses
```

Comment:
This comment should be explaining why we are doing what we are doing, not what we are doing....  Why is this the right thing to check for?  Do we not care about IP addresses for operational browse?  If so, _that_ is what we should document here.